composer.json

```diff
@@ -13,13 +13,21 @@
     }
   ],
   "require": {
-    "typo3/cms-core": "^9.5|^10.4",
+    "typo3/cms-core": "^9.5.20|^10.4.6",
```

Comment:
I suggest leaving v9 out of this. Leaving it in would mean we can't use PSR-14 events.